src/tv2-common/__tests__/jinglePartProperties.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { GetJinglePartPropertiesFromTableValue } from 'tv2-common'
+
+describe('GetJinglePartPropertiesFromTableValue', () => {
+	it('Calculates values correctly', () => {
```

Comment:
When testing calculations (and just in general) it's a good idea to have multiple unit tests with different inputs to ensure your calcalations are correct and not just happens to work for this one specific input.
It's also a good idea to have test with bad inputs to ensure your methods handles it correctly.